Paintroid/src/main/java/org/catrobat/paintroid/presenter/MainActivityPresenter.java

```diff
@@ -419,30 +414,30 @@ public void finishInitialize() {
 			exitFullscreen();
 		}
 
-		if (model.isOpenedFromCatroid()) {
-			navigationDrawerViewHolder.removeItem(R.id.pocketpaint_nav_save_image);
-			navigationDrawerViewHolder.removeItem(R.id.pocketpaint_nav_save_duplicate);
-			navigationDrawerViewHolder.removeItem(R.id.pocketpaint_nav_new_image);
-		} else {
-			navigationDrawerViewHolder.removeItem(R.id.pocketpaint_nav_back_to_pocket_code);
-			navigationDrawerViewHolder.removeItem(R.id.pocketpaint_nav_export);
-			navigationDrawerViewHolder.removeItem(R.id.pocketpaint_nav_discard_image);
-		}
-		navigationDrawerViewHolder.setVersion(BuildConfig.VERSION_NAME);
-
 		view.initializeActionBar(model.isOpenedFromCatroid());
 
 		if (!commandManager.isBusy()) {
 			navigator.dismissIndeterminateProgressDialog();
 		}
 	}
 
+	@Override
+	public void removeMoreOptionsItems() {
+		if (model.isOpenedFromCatroid()) {
+			topBarViewHolder.removeItem(R.id.pocketpaint_nav_save_image);
```

Comment:
Please let topbarviewholder remove the items

Reply:
e.g. `topBarViewHolder.removeStandaloneMenuItems(Menu menu)` and`topBarViewHolder.removeCatroidMenuItems(Menu menu)`